installGitHooks.bat

```diff
@@ -0,0 +1 @@
+./gradlew installGitHooks
```

Comment:
Shouldn't this be `./gradlew.bat` instead of `./gradlew`?  Besides, do we need this `bat` file for just a single command?

Reply:
you don't need the bat extension but I'll remove it. I just added it to have an easy way to install the hooks without looking up the command again

Reply:
removed! 👍